kafka/event.go

```diff
@@ -145,6 +145,33 @@ func (o OAuthBearerTokenRefresh) String() string {
 	return "OAuthBearerTokenRefresh"
 }
 
+// NewApplicationRebalanceError reports an application-level error occuring during a RebalanceCb call.
+func NewApplicationRebalanceError(wrapped error) *ApplicationError {
+	return &ApplicationError{
+		Err: fmt.Errorf("rebalance error: %w", wrapped),
+	}
+}
+
+// ApplicationError reports an error triggered by the application and
+// returned by this package as en event. One such case exists with application
+// RebalanceCb error results.
+type ApplicationError struct {
+	Err error
```

Comment:
This could be a private field since there's Unwrap() and the other accessor methods, right?

Reply:
Right, done.